src/main/java/core/model/player/Player.java

```diff
@@ -466,24 +467,41 @@ public Player(java.util.Properties properties, HODateTime hrfdate, int hrf_id) {
             m_bTrainingBlock = oldPlayer.hasTrainingBlock();
             motherclubId = oldPlayer.getMotherclubId();
             motherclubName = oldPlayer.getMotherclubName();
-            if (motherclubId == null) {
-                var playerDetails = OnlineWorker.downloadPlayerDetails(this.getPlayerID());
-                if (playerDetails != null) {
-                    motherclubId = playerDetails.getMotherclubId();
-                    motherclubName = playerDetails.getMotherclubName();
-                }
-            }
         }
     }
 
     public String getMotherclubName() {
+        DownloadMotherclubInfoIfMissing();
         return this.motherclubName;
     }
 
     public Integer getMotherclubId() {
+        DownloadMotherclubInfoIfMissing();
         return this.motherclubId;
     }
 
+    private void DownloadMotherclubInfoIfMissing() {
```

Comment:
This should be a lowercase `D` at the start.

Reply:
yep - i was Dreaming